release/index.d.ts

```diff
@@ -0,0 +1,9 @@
+import { DataType } from "./src/datatype";
```

Comment:
This is autogenerated. Isn't it fine, since the only thing that would ever read this is TypeScript?